web/client-api/src/main/java/io/deephaven/web/client/api/WorkerConnection.java

```diff
@@ -759,14 +759,68 @@ public Promise<JsTable> getTable(JsVariableDefinition varDef, @Nullable Boolean
         });
     }
 
+    /**
+     * Export a table from the provided ticket
+     * 
+     * @param ticket Ticket of the table to export
+     * @param fetchSummary Description for the fetch. Used for logging and errors.
+     * @return Promise for the table
+     */
+    public Promise<JsTable> getTable(TypedTicket ticket, String fetchSummary) {
```

Comment:
maybe a name that is clear about how this is intended for server-side exports only (i.e. note most table tickets)? the api would totally let you pass in a plain typedticket and summary, but would fail for client tickets/etc.

Reply:
Updated names